src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -230,8 +236,10 @@ def _append_trait_val(
     if data_id not in traits_val:
         traits_val[data_id] = list()
 
+    detected_gene_score = json.loads(detected_gene_score.replace("'", '"'))
```

Comment:
why is detected gene score wrapped in `'`s, and why does it need to be wrapped in `"`s before deserializing?

Reply:
it's how dataframe saves strings dict - everything is single quoted. I changed it to use `ast.literal_eval` if that makes things less confused.

Reply:
Hmm, would it make more sense just to dump the gene score dict as json in the calculation script?

Reply:
👍 sure.

Reply:
:+1: